src/App.tsx

```diff
@@ -26,70 +27,127 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  if (sortType !== SortType.NONE) {
+    visibleGoods.sort((good1, good2) => {
+      switch (sortType) {
+        case SortType.ALPHABET:
+          return good1.localeCompare(good2);
+
+        case SortType.LENGTH:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
```

Comment:
Check sort type before sort launching. 
Delete if statement, you have default case inside switch which do the same

```suggestion
      switch (sortType) {
        case SortType.ALPHABET:
          visibleGoods.sort((good1, good2) => (
           good1.localeCompare(good2);
         )
         
       case ...
```